handlers/main.yaml

```diff
@@ -10,3 +10,14 @@
     name: redis
     state: restarted
   when: not (redis_service_result is defined and redis_service_result.changed)
+
+- name: Start Redis Sentinel
+  service:
+    name: redis
+    state: started
+  register: redis_sentinel_service_result
```

Comment:
Variable name also used in `tasks/install.yaml` but this `register` does not seem necessary?

Reply:
This is still an artifact from the service starting at installation. Will remove it.